ui/ui.gradle.kts

```diff
@@ -187,80 +201,88 @@ tasks.register<Copy>("collectDependencies") {
     into(buildDir.resolve("installerInput"))
 }
 
-/**
- * Build a Java 11 runtime image with jlink. Otherwise a Java 13 image would be generated by
- * jpackage, which is not ideal given that Java 13 is not yet stable.
- *
- * This task regenerates the jlink image every time it is invoked.
- */
-tasks.register<Exec>("jlink") {
-    group = "Installer generation"
-    description = "Generates a runtime image for a native installer."
-
-    val outputDir = project.buildDir.resolve("jlink").absolutePath
-    outputs.file(outputDir)
-    delete(outputDir)
-
-    setCommandLine(
-            Jvm.current().javaHome.resolve("bin/jlink").absolutePath,
-            "--add-modules",
-            "java.base,java.desktop,java.datatransfer,java.logging,java.management,java.naming,java.prefs,java.scripting,java.sql,java.xml,jdk.dynalink,jdk.scripting.nashorn,jdk.unsupported",
-            "--output", outputDir,
-            "--no-header-files",
-            "--compress=2",
-            "--no-man-pages",
-            "--strip-debug"
-    )
-}
+if (BuildType.isJdk11) {
+    /**
+     * Build a Java 11 runtime image with jlink. Otherwise a Java 13 image would be generated by
+     * jpackage, which is not ideal given that Java 13 is not yet stable.
+     *
+     * This task regenerates the jlink image every time it is invoked.
+     */
+    tasks.register<Exec>("jlink") {
+        group = "Installer generation"
+        description = "Generates a runtime image for a native installer."
+
+        val outputDir = project.buildDir.resolve("jlink").absolutePath
+        outputs.file(outputDir)
+        delete(outputDir)
+
+        setCommandLine(
+                Jvm.current().javaHome.resolve("bin/jlink").absolutePath,
+                "--add-modules",
+                "java.base,java.desktop,java.datatransfer,java.logging,java.management,java.naming,java.prefs,java.scripting,java.sql,java.xml,jdk.dynalink,jdk.scripting.nashorn,jdk.unsupported",
```

Comment:
Can you turn this into a variable as a list and then when its used, `joinToString`?
That way, if we need to add more, it's easier/clearer what's been added/removed.

Reply:
Makes sense